bounties/level-1/claimable_balances_account_viewer.md

```diff
@@ -0,0 +1,38 @@
+# Pending Claimable Balances in the Stellar Account Viewer
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 1000 XLM | 500 XLM | Competitive | 
+
+## Description
+The stellar account viewer is a great tool that allows you to see how many XLM you own, the transactions you received and the option to send payments.
```

Comment:
Since the account viewer is so central to this bounty, we might want to have links to the account viewer website and repository up here, as well.